package.json

```diff
@@ -40,9 +40,9 @@
     "istanbul-reports": "^2.2.6",
     "rimraf": "^3.0.0",
     "test-exclude": "^5.2.3",
-    "v8-to-istanbul": "^3.2.6",
-    "yargs": "^15.0.0",
-    "yargs-parser": "^16.0.0"
+    "v8-to-istanbul": "4.0.0",
+    "yargs": "^14.0.0",
```

Comment:
whoops, looks like I screwed up the merge of `package.json`, mind setting this back to `15.0.0` and `16.0.0`.